.bundle/config

```diff
@@ -1,4 +1,5 @@
 ---
 BUNDLE_PATH: "vendor/bundle"
-BUNDLE_JOBS: "16"
+BUNDLE_JOBS: "3"
 BUNDLE_WITHOUT: "screenshots"
+BUNDLE_RETRY: "3"
```

Comment:
Don't think we need to change this, can we revert this?

Reply:
Yeah, I'll revert it, not sure how that got there.